primary/src/helpers/i18n.ts

```diff
@@ -271,7 +273,8 @@ function qqxOutput(outputWrapper = (output: string) => output) {
 		descriptor.value = function (this: I18n, query: string, parameters?: I18nParameters) {
 			if (this.locale === 'qqx') {
 				return outputWrapper(this.sanitizeHtml(`${func}(${query}${parameters && Object.keys(parameters).length !== 0 ? ', ' + JSON.stringify(parameters) : ''})`));
-			} else {
+			}
```

Comment:
are these single character lines really necessary? imo it looked nicer when the conditional blocks were all together

Reply:
IMO the Stroustroup brace style (https://eslint.org/docs/latest/rules/brace-style) is a nice balance between being condensed and being easy to read. The advantage of Stroustroup is that you can leave a comment above your `else` and `else if` statements if necessary. It's good to keep things consistent, and you're able to configure Code/ESLint to autofix these code style rules when you save.